scripts/gen_tiny_gguf.py

```
@@ -0,0 +1,301 @@
+        header.extend(struct.pack('<Q', offset))
+
+    # Align header to 32 bytes
+    alignment = 32
+    header_size = len(header)
+    aligned_header = ((header_size + alignment - 1) // alignment) * alignment
+    padding = aligned_header - header_size
+    header.extend(bytes(padding))
+
+    # Write data section
+    data_section = bytearray()
+    for i, (name, dtype, dims, data) in enumerate(tensors):
```

Ruff (0.16.1)

[warning] Loop control variable `name` not used within loop body

(B007)